out_writeapi.go

```diff
@@ -317,7 +327,8 @@ func sendRequestRetries(ctx context.Context, data [][]byte, config **outputConfi
 func sendRequestDefault(ctx context.Context, data [][]byte, config **outputConfig, streamIndex int) error {
 	(*config).mutex.Lock()
 	defer (*config).mutex.Unlock()
-	currStream := (*(*config).managedStreamSlice)[streamIndex]
+	streamSlice := *(*config).managedStreamSlice
+	currStream := streamSlice[streamIndex]
```

Comment:
Since streamSlice is not used after this, is it still necessary to break the original 1 line into these 2 lines?